ServerCore/Pages/Teams/Play.cshtml

```diff
@@ -55,25 +55,24 @@
             <td>
                 @Html.DisplayFor(modelItem => item.Puzzle.PuzzleFile.ShortName)
             </td>
-    @if (showAnswers) {
             <td>
-                @Html.DisplayFor(modelItem => item.Puzzle.AnswerFile.ShortName)
+                <a asp-page="/Submissions/Index" asp-route-puzzleid="@item.Puzzle.ID" asp-route-teamid="@Model.TeamID">@(item.State.IsSolved ? "See Answers" : "Submit Answer")</a>
             </td>
-    }
+            @if (showAnswers) {
+                <td>
+                    @Html.DisplayFor(modelItem => item.Puzzle.AnswerFile.ShortName)
+                </td>
+            }
             <td>
-                @* TODO: link to Jessica's answer submission page here *@
-                <a asp-page="./Play" asp-route-id="@item.Puzzle.ID">
```

Comment:
Information seems to be jumping between table columns here, without a matching change to the table headers. Is this a merge issue perhaps? I would have expected you to move the /Submissions/Index link to this line and remove the comment but you seem to have made far more extensive changes than that.